files/brews/zlib.rb

```diff
@@ -3,7 +3,7 @@
 class Zlibphp < Formula
   homepage 'http://www.zlib.net/'
   url 'http://zlib.net/zlib-1.2.8.tar.gz'
```

Comment:
@typhonius FYI this url is now dead 

```
$ curl --HEAD http://zlib.net/zlib-1.2.8.tar.gz
HTTP/1.1 404 Not Found
```

Fixed in https://github.com/AmpersandHQ/puppet-php/pull/2